help/faq/texprobs.md

```diff
@@ -0,0 +1,88 @@
+# Why doesn't my processed TeX submission look the way I expected it?
+
+There are a number of common reasons why the processed version of TeX submissions to arXiv may not appear the same as it does when processed locally. In most cases there are fixes that can be applied and the submission replaced to correct the problems.
+
+*   [Lines in the table of contents don't wrap](#toc_links)
+*   [Fuzzy fonts in PDF](#fuzzy_pdf)
+*   [PostScript figures lose quality in the arXiv-generated PDF](#distiller_params)
+*   [Extra white space around figures covers up some of the text](#powerpoint)
+*   [Why don't the labels for my figures appear when I use Feynmf?](feynmf#nolabel)
```

Comment:
Broken link, found at faq/feynmf

Reply:
curious, because it looks like feynmf.md exists
https://github.com/arXiv/arxiv-docs/blob/develop/help/faq/feynmf.md#nolabel

Reply:
I see what the issue is: both /help/texprobs and /help/faq/texprobs exist and I was looking at the non-faq nested one.